frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -1,14 +1,29 @@
 import React from "react";
 import { Card } from "react-bootstrap";
+import moment from "moment";
```

Comment:
I've asked that the dependency on `moment.js` be removed.

See: https://github.com/ucsb-cs156-w22/team04-w22-5pm-HappyCows/issues/59